Automattic/jetpack
# Internationalization: remove all custom language handling and let language packs do the work

#### Changes proposed in this Pull Request:

* Remove locale info from the Initial_State JSON that's loaded with the React app, now that it will be loaded by WordPress.
* Remove function used to build the above.
* Remove Gulp tasks used to 1) extract strings from the React codebase into a _inc/jetpack-strings.php file (languages:extract), and then 2) create .json files from the .po files we get from translate.wordpress.org (languages:build)
* Move to language packs for all translation handling in Jetpack.
* Remove all language files

Primary issue: #16481 
Internal reference: p9dueE-1D0-p2

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* N/A

#### Testing instructions:

* Go to Settings > General and change your site language to French.
* Go to Dashboard > Updates and update translations.
* The React dashboard will remain in English (for now), but should load properly.
* Other Jetpack features (such as options on wp-admin pages, widgets, ...) should be translated properly.

#### Proposed changelog entry for your changes:

* N/A
